.github/CONTRIBUTING.md

```diff
@@ -71,22 +108,75 @@ Documentation for running and updating our existing tests, as well as the code f
+- Explain what testing has been done on the PR and what may be left to do. Screenshots, gif, video and text instructions documenting the tests can be very useful. Those tell other contributors looking at the PR how to verify the changes. "This was tested in WordPress" **is not enough** to meet our review criteria, see [Review Criteria](#review-criteria) above.
+- Look to make sure the backport is done correctly, with attribution to the original WP committer. 
+- If there are merge conflicts during the backport, check if they have been resolved correctly by comparing the final changes from the full PR against the corresponding WP changeset(s). Remember to fix these **in a separate commit**. If you cannot resolve the conflicts, ask for help in the [Core slack channel](https://www.classicpress.net/join-slack/) or just push your branch as-is and we'll take care of it!
 
-If you're not sure about any of that, then it's a good idea to ask first. A good way is to create an issue for the specific change you're interested in, along with links to the relevant WordPress changesets and tickets, and any other information you have about how the change works.
+### Making a Backport PR.
+When you're ready to backport a code change, make a new branch to your ClassicPress fork using:
+```
+git checkout upstream/develop -B branch-name
+```
+
+See [how to set up local environment](#setting-up-local-testing-and-dev-environment).
+
+1. Identify the WordPress **changeset number** that you'd like to port such as `43123`.
+2. Run `bin/backport-wp-commit.sh` script in your terminal/command prompt.
+
+```
+bin/backport-wp-commit.sh CHANGESET_NUMBER
+```
+ to apply the change to the code. 
+ 
+ If you're porting multiple changesets, you can use the `-c` option to this script 
 
-Finally, when you're ready to backport a code change, identify the WordPress **changeset number** that you'd like to port, and run the `bin/backport-wp-commit.sh` script to apply the change to the code. If you're porting multiple changesets, you can use the `-c` option to this script to apply all the changesets to the same branch.
+```
+bin/backport-wp-commit.sh -c CHANGESET_NUMBER
+```
+to apply all the changesets to the same branch. 
 
-Using this script for all backports saves time for you and for the maintainers, and it also uses a standardized format for commit messages which makes it possible for us to track which WordPress changes we've already included.
+3. Resolve merge conflicts (if any)
+4. Repeat steps 2 and 3
+5. Then push your branch to your fork online using:
+   
+```
+git checkout upstream/develop -B desired-branch-name
+```
+6. Use the online platform to make a PR to the `develop` branch for review.
+!["Opening a PR to the develop branch"][open_pr_br_develop]
```

Comment:
This is not the right image. Maybe let's leave that out for now?

Reply:
Will revert this. Should I squash changes?

Reply:
No need to squash changes, they will get rolled up into one commit when we merge this.